engine/servermaster/jobmanager.go

```diff
@@ -613,3 +646,33 @@ func (jm *JobManagerImpl) WatchJobStatuses(
 	receiver := jm.notifier.NewReceiver()
 	return snapshot, receiver, nil
 }
+
+func (jm *JobManagerImpl) bgJobOperatorLoop(ctx context.Context) {
+	jm.wg.Go(func() error {
+		timer := time.NewTimer(jobOperateInterval)
+		for {
+			select {
+			case <-ctx.Done():
+				return errors.Trace(ctx.Err())
+			case <-jm.jobOperatorNotifier:
+			case <-timer.C:
+			}
+			if err := jm.jobOperator.Tick(ctx); err != nil {
+				inErr := jm.Exit(ctx, framework.ExitReasonFailed, err, "")
```

Comment:
Why we expect the whole jobmanager exit when job operator is fail?
What about keep log error and try?

Reply:
addressed